extensions/GlobalMessages/tests/GlobalMessages0.data.php

```diff
@@ -0,0 +1,970 @@
+// 45 --------------------------------------------------------------------------
+'emailext-emailconfirmation-changed-footer-2' => 'See you on Fandom!',
+// 46 --------------------------------------------------------------------------
+'emailext-emailconfirmation-changed-subject' => 'Confirm your email address change on Fandom',
+// 47 --------------------------------------------------------------------------
+'emailext-emailconfirmation-changed-summary' => 'Please click the link below to confirm your change of email address on Fandom, so you can keep getting everything to fuel your fandom.',
+// 48 --------------------------------------------------------------------------
+'emailext-emailconfirmation-community-team' => 'The Fandom Community Team',
+// 49 -REMOVED / SOCIAL TEAM ---------------------------------------------------
+// 'emailext-emailconfirmation-footer-2' => 'Can\'t wait to see you in the community!',
+// 50 --------------------------------------------------------------------------
+'emailext-emailconfirmation-reminder-summary' => 'It\'s been a few days, but it looks like you haven\'t confirmed your account on Fandom yet. We can\'t wait for you to fully dive into the Fandom universe! Just click the confirmation link below and you\'ll be good to go:',
+// 51 IS A DUPLICATE OF 50 -----------------------------------------------------
+// 'emailext-emailconfirmation-reminder-summary' => 'It\'s been a few days, but it looks like you haven\'t confirmed your account on Fandom yet. We can’t wait for you to fully dive into the Fandom universe! Just click the confirmation link below and you’ll be good to go:',
+// 52 --------------------------------------------------------------------------
+'emailext-emailconfirmation-subject' => 'Confirm your email and get started on Fandom!',
+// 53 --------------------------------------------------------------------------
+'emailext-emailconfirmation-summary' => 'You\'re SO CLOSE! Click the link below to confirm your email address and get started on Fandom:',
+// 54 IS A DUPLICATE OF 53 -----------------------------------------------------
+// 'emailext-emailconfirmation-summary' => 'You’re SO CLOSE! Click the link below to confirm your email address and get started on Fandom:',
+// 55 --------------------------------------------------------------------------
+'emailext-fanverse-tagline' => 'Powered by Wikia',
+// 56 --------------------------------------------------------------------------
+'emailext-fbdisconnect-instructions' => 'If you already set a password on Fandom, don\'t worry! Your account is safe and secure. You can ignore this email and continue signing in to Fandom with your old password. Otherwise, please sign in with the temporary password above and reset it to your password of choice on your Preferences page.',
+// 57 --------------------------------------------------------------------------
+'emailext-fbdisconnect-subject' => 'Your Fandom account is now disconnected from Facebook!',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Your Fandom account is now disconnected from Facebook!" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)